lua/bufferline/highlight.lua

```diff
@@ -60,10 +60,11 @@ hl.set_default_link('BufferVisibleTarget', 'BufferDefaultVisibleTarget')
 hl.set_default_link('BufferVisibleWARN', 'BufferDefaultVisibleWARN')
 
 -- NOTE: these should move to `setup_defaults` if the definition stops being a link
-hl.set_default_link('BufferDefaultCurrentIcon', 'BufferDefaultCurrent')
-hl.set_default_link('BufferDefaultInactiveIcon', 'BufferDefaultInactive')
-hl.set_default_link('BufferDefaultVisibleIcon', 'BufferDefaultVisible')
-hl.set_default_link('BufferDefaultOffset', 'BufferDefaultTabpageFill')
+hl.set_default_link('BufferDefaultAlternateIcon', 'BufferAlternate')
```

Comment:
These should be linked to something that _might_ link to a default, but also could be user-defined.